app/steps/ui/declaration/template.html

```diff
@@ -63,8 +63,9 @@ <h2 class="heading-large declaration-subheader">{{ content.declarationHeader }}<
     </div>
     <div class="declaration-grid-item">
         <h2 class="heading-medium">{{ content.saveThisPage }}</h2>
-        <p><img class="icon-file-download" alt="Download PDF document"/>&nbsp;
-            <a id="declarationPdfHref" href="/declarationPdf">{{content.declarationPdf}}</a></p>
+        <p><img src="{{ asset_path }}images/icon-file-download.png?" alt="Download pdf" width="30" height="30"/>&nbsp;
```

Comment:
We don't need to add this image manually as the GDS toolkit has a class that can add it so we should be using that instead

Reply:
yes, templtae has already been changed to use <i class="icon icon-file-download"></i>